LondonBritishLibrary/orient/BLorient8814.xml

```diff
@@ -0,0 +1,538 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="BLorient8814" xml:lang="en" type="mss">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Mazmura Dǝngǝl, Wǝddāse Māryām, ʾAnqaṣa bǝrhān, Sǝbḥata Dǝngǝl, prayers and hymns, genealogy of Ethiopian monks,
+                    Maṣḥafa śǝrʿat, Malkǝʾ of the Trinity, Miracles of the Virgin Mary</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="CH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                    Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                        licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS00001BL"/>
+                        <collection>Oriental</collection>
+                        <idno>BL Oriental 8814</idno>
+                        <altIdentifier><idno>Strelcyn 32</idno></altIdentifier>
+                        <altIdentifier><idno>Curzon, Cat. E7</idno></altIdentifier>
+                        <altIdentifier><idno>Parham 61</idno></altIdentifier>
+                    </msIdentifier>
+                    <msContents>
+                        <msItem xml:id="ms_i1">
+                            <locus from="1ra" to="36rb"/>
+                            <title ref="LIT3985Mazmura" type="incomplete"/>
+                            <msItem xml:id="ms_i1.1">
+                                <title type="incomplete">Interpolations to the Psalms</title>
```

Comment:
So did I, wondering what 'interpolations to the Psalms' might be. For the record, I stick to what Strelcyn noted in his description. If I understand him correctly, these are parts of the text that are often inscribed in psalters as interpolations between the psalms or other parts of the psalter, but in this case they are collected and kept separate in this manuscript. But I could be wrong.
![Screenshot 2024-03-26 113826](https://github.com/BetaMasaheft/Manuscripts/assets/40291787/a1677146-8011-422c-8c7d-89288964402d)

Reply:
Thank you. I would interpret the remark on interpolation as Strelcyn's description of the work, not strictly necessary for us, since we identify the work with its ID. It is interesting, if this manuscript really contains only the Marian hymns of the Mazmura dǝngǝl, it would be according to my knowledge the only known witness containing this work without the Psalter. Maybe @DariaElagina has opinions on this.